### DISCUSSION TITLE
Error Student Pack approve

### DISCUSSION THREAD

**Discussion by Matyu9** [community/community#147854] [GitHub Education] [answered]
### Hi everyone,

Hi everyone! 
Last year (Dec 3, 2023) I have been approved to the program 🥳. And I used the Jetbrains License for their IDE.
However, the end of this license was on Dec 14, 2024. I would have liked to renew it, but my Github Education can't be renewed somehow...
![Screenshot 2024-12-24 at 10 16 23](https://github.com/user-attachments/assets/5542f14a-3662-4f5b-a946-76ef0aca91ba)
If someone has an idea, I will try it... 
Thanks for all.
Matyu

**Comment by Ravindulakmina** [comment]
Isn't there a reapply button there?

**Reply by Matyu9** [reply]
There is a button but it's not available...
![Screenshot 2024-12-24 at 10 25 52](https://github.com/user-attachments/assets/eb77664a-075d-470c-b0e0-969c6da354ad)

**Reply by Ravindulakmina** [reply]
You already got it.
After applying for the second time, we cannot enjoy some things. If you look closely at the student pack, you can see that some things are missing.

**Reply by Matyu9** [reply]
Yeah I know but this time I cannot reapply Github Education Pack

**Reply by Ravindulakmina** [reply]
If you feel there is a problem, I think you should contact github support.
I also get the same yellow message above. I can't apply either. The reason is that I have already applied it a second time.

**Reply by Matyu9** [reply]
It's maximum 2 times per account ?

**Reply by Ravindulakmina** [reply]
I am not aware of that. I think it changes depending on our confirmation.

**Reply by Matyu9** [reply, marked as answer]
Ok I will open a ticket on the support thanks !

**Reply by Ravindulakmina** [reply]
Yes, do it.
